mozci/push.py

```diff
@@ -32,7 +32,7 @@
 
 BASE_INDEX = "gecko.v2.{branch}.revision.{rev}"
 
-MAX_DEPTH = 14
+MAX_DEPTH = config.get("maxdepth", 20)
```

Comment:
Was going to say you should add this config option to the docs.. but we don't have configuration page in the docs.. oops. I'll file an issue.